src/test/java/org/jenkinsci/plugins/workflow/cps/FlowDurabilityTest.java

```diff
@@ -865,19 +877,17 @@ public void evaluate() throws Throwable {
             @Override
             public void evaluate() throws Throwable {
                 WorkflowRun run = story.j.jenkins.getItemByFullName(jobName, WorkflowJob.class).getLastBuild();
+                if (run == null) {
+                    return;
```

Comment:
Is this expected?

Reply:
Happened in testing -- if the fuzzer kills things *too* early the Run may not even have had time to be created yet (rare but happens).

Reply:
Probably deserves a comment to that effect.

Reply:
(resolved)